runtimes/super-runtime/src/lib.rs

```diff
@@ -366,6 +366,12 @@ impl system::offchain::CreateTransaction<Runtime, UncheckedExtrinsic> for Runtim
 
 // end of offchain_demo setting
 
+impl randomness::Trait for Runtime {
+	type Event = Event;
+	type CollectiveFlipRandomnessSource = RandomnessCollectiveFlip;
+	type BabeRandomnessSource = Babe;
```

Comment:
There is afaik never any reason to have both randomness sources in one module.  Always use BABE if you need security.  Use RandomnessCollectiveFlip only for tests where BABE is not available.

Reply:
Totally agree there is no need for both in the wild. They are both used here merely to show how either would be used. I'll make that more clear in the comments.